src/pages/UserInfo.vue

```diff
@@ -0,0 +1,75 @@
+<template>
+  <v-flex xs12 sm6 offset-sm3>
+    <v-card>
+      <v-card-title class="headline">
+        My Account
+      </v-card-title>
+      <v-card-text>
+        <table class="user-info-table" v-if="user">
```

Comment:
Visst fanns det mer data va? Eller kanske bara det här man ser med email/pwd?

Reply:
Jo, det finns det
https://firebase.google.com/docs/reference/js/firebase.UserInfo
Men vi använder bara DisplayName (we set it when the user signs up). Skulle kunna koppla in mer sen.